### ISSUE TITLE
Contributing in nuclei segmentation using an external library

### ISSUE THREAD

**Issue by ajinkya-kulkarni** [DigitalSlideArchive/HistomicsTK#981]
Hi Devs, thanks for this repo!
Are contributions to the nuclei segmentation part using an external library allowed? For example using CellPose or something similar.

**Comment by AhmedElshahedd** [comment]
Hello,
I have a question regarding how to build my own analysis using models developed with PyTorch? (eg. patch based classification)

**Comment by manthey** [comment]
@AhmedElshahedd There are so many approaches that the best one depends on what you are trying to accomplish.  For instance, if you are trying to analyze whole slide images at full resolution a patch at a time, the methods used in HistomicsTK are appropriate (a docker with torch installed and using a tile iterator to process each patch).  If you want to only process user selected patches quasi realtime, Monai Label or https://github.com/DigitalSlideArchive/dsa-run-custom-ai-models would be more appropriate.  See also https://github.com/DigitalSlideArchive/superpixel-classification for a way that works on multiple images and uses tensorflow (but there is a branch that is in progress that uses torch).
